config/locales/en.yml

```diff
@@ -390,6 +390,23 @@ en:
       user: "User"
       liked: "Liked"
       action: "Action"
+  map:
+   index:
```

Comment:
Hi @amiyahfrierson and thanks for working on this.
It looks like the indentation is not correct somewhere. The indentation  should be is 2 spaces
Could you please confirm if `map`, `index` and `current_view` are on the same line as the `like`, `index` and `recent_likes`  on line 387,388 and 389 respectively?.Thanks

Reply:
@cesswairimu Made the changes! Hopefully it's fixed.